arcgis-ios-sdk-samples/Maps/Display a map SwiftUI/SwiftUIMapView.swift

```diff
@@ -0,0 +1,88 @@
+// Copyright 2021 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+import ArcGIS
+
+struct SwiftUIMapView {
```

Comment:
> I'm guessing, based on how they are written, that the types in this file aren't intended to be reused by multiple samples. Therefore they should be made private.

> ... implementation can be more easily copied-and-pasted into another sample

Discussed with Divesh and he said this will be the only SwiftUI sample in the current sample viewer. Thus I'll keep them `internal` and in this separate file, so that the file structure is clearer.